packages/core/core-flows/src/product/steps/normalize-products-v1.ts

```diff
@@ -0,0 +1,70 @@
+import { HttpTypes } from "@medusajs/framework/types"
+import {
+  CSVNormalizer,
+  Modules,
+  productValidators,
+} from "@medusajs/framework/utils"
+import { StepResponse, createStep } from "@medusajs/framework/workflows-sdk"
+import { convertCsvToJson } from "../utils"
+
+/**
+ * The CSV file content to parse.
+ */
+export type NormalizeProductCsvV1StepInput = string
+
+export const normalizeCsvV1StepId = "normalize-product-csv-v1"
```

Comment:
This might not be the best naming. So feel free to send some recommendations

Reply:
question: should we remove V1 from the name?